src/componets/Orderform.jsx

```diff
@@ -0,0 +1,59 @@
+
+function Orderform({fruitCounts}) {
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+
+        console.log(fruitCounts)
+
+        const formData = new FormData(e.target);
+        formData.forEach((value, key) => {
+            console.log(`${key}: ${value}`);
+        });
+    };
+    return (
+        <div>
+            <form onSubmit={handleSubmit} method="POST">
+                <label htmlFor="firstname">Voornaam:</label>
+                <input type="text" id="firstname" name="firstname" required/>
+                <br/><br/>
```

Comment:
`<br/> ` is niet heel netjes om te gebruiken, hier kan je het beste padding of margin voor gebruiken!